packages/tslint/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "@calcom/tslint",
```

Maybe? To avoid confusion with [TypeScript Lint](https://github.com/palantir/tslint)
```suggestion
  "name": "@calcom/localelint",
```

---

Sounds good 👍 I renamed this line as well as the directory's name